dist/expanding.jquery.js

```diff
@@ -194,8 +194,8 @@ function Expanding (textarea) {
   this.element.appendChild(this.textareaClone.element)
 
   var inputHandler = this.update.bind(this)
-  this.textarea.on(inputEvent, inputHandler)
-  this.textarea.on('change', inputHandler)
+  this.textarea.on(inputEvent, this.update.bind(this))
+  if (inputEvent !== 'input') this.textarea.on('paste', inputHandler)
```

Comment:
I'd say either use inputHandler in both places or this.update.bind(this) in both places.

Also, 'change' seems a bit more robust vs paste - I was just bringing the paste context menu as an example - there are probably other cases where the input changes but a paste didn't happen (for instance, dragging text around).